po/ca.po

```diff
@@ -156,358 +191,372 @@ msgid ""
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:123
 msgid "New approval dialogs for external apps, e.g. appstream:// or ftp://"
-msgstr ""
+msgstr "Nous diàlegs de confirmació per aplicacions externes, p.e. appstream:// o ftp://"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:124
 msgid "New warning when no network connection"
-msgstr ""
+msgstr "Nou avís quan no hi ha connexió a la xarxa"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:125
 msgid "Prettier errors using native views"
-msgstr ""
+msgstr "Errors més bonics utilitzant les vistes natives"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:126
 msgid "Stopped throwing away text when unfocusing URL entry"
-msgstr ""
+msgstr "S'ha deixat de llançar text quan l'entrada d'URL perd el focus"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:127
 msgid "Stopped searching when the URL entry is empty"
-msgstr ""
+msgstr "S'ha deixat de cercar quan l'entrada d'URL està buida"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:128
 msgid "Code cleaning to keep things smelling fresh"
-msgstr ""
+msgstr "Neteja de codi per mantindre les coses amb olor a fresc"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:134
 msgid "Fresh styles to keep things funky, plus more:"
-msgstr ""
+msgstr "Nous estils per mantindre les coses a la moda, i a més:"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:136
 msgid ""
 "The new start page makes Ephemeral faster and makes it not touch the network "
 "until you're ready to browse"
 msgstr ""
+"La nova pàgina d'inici fa Ephemeral més ràpid i que no toque la xarxa "
+"fins que estigueu preparats per navegar"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:137
 msgid "Middle- or Ctrl-click a link to open it in a new window"
-msgstr ""
+msgstr "Féu Ctrl-click o click amb el botó del mig per obrir un enllaç a una nova finestra"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:138
 msgid ""
 "Displayed URLs are kept in sync for pages that manually update the URL "
 "without a page load"
 msgstr ""
+"Les URLs mostrades es mantenen sincronitzades per aquelles pàgines que actualizen manualment "
+"la URL sense carregar la pàgina"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:139
 msgid "Local HTML files can now be opened"
-msgstr ""
+msgstr "Ara es poden obrir els fitxers HTML locals"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:146
 msgid "More informative \"set as default\" infobar"
-msgstr ""
+msgstr "Barra \"establiu per defecte\" més informativa"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:147
 msgid "Tweaked \"set as default\" infobar styling"
-msgstr ""
+msgstr "S'ha modifica l'estil de la barra \"establiu per defecte\""
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:154
 msgid "Search from URL entry!"
-msgstr ""
+msgstr "Cerquey des de l'entrada d'URL!"
```

Comment:
Little slip of the finger here (I myself am very prone to this typo :stuck_out_tongue: )